programs/models.py

```diff
@@ -223,14 +234,15 @@ def from_model_data(self, data: DataType):
         program_category = self.instance
 
         program_category.calculator = data["calculator"]
-        program_category.icon = data["icon"]
```

Comment:
@CalebPena @msrezaie Removing this line is the only "real" change in this file. I think this line duplicates the logic on [248](https://github.com/Gary-Community-Ventures/benefits-api/blob/5a0b71e243c18baac7666059e39e481a1f43d2c3/programs/models.py#L248-L254)